product_docs/docs/pgd/5/admin-tpa/index.mdx

```diff
@@ -22,3 +22,5 @@ This section of the manual covers how to use TPA to deploy and administer EDB Po
   * Deploying the configuration with TPA
 
 The installing section provides an example cluster which will be used in future examples.
+
+You can also [perform a rolling major version upgrade](upgrading_major_rolling.mdx) with PGD administered by TPA.
```

Comment:
We don't use words like "guide," "section," "document," etc, as these are "documenting the documentation." I think I fixed the rest of the file when editing DJ's stuff but I'm not sure...